src/components/Toast.tsx

```diff
@@ -55,10 +56,17 @@ const Toast = ({
   swipeable,
   position = 'bottom',
   onDismiss,
+  onUnmount,
```

Comment:
forwarded from `BottomSheetInlineNotification` to handle the edge case when user navigates from the screen without explicitly dismissing the toast.

context: https://github.com/valora-inc/wallet/pull/5045/files#r1513591182